src/config/syscheck-config.c

```diff
@@ -442,6 +454,13 @@ static int read_attr(syscheck_config *syscheck, const char *dirs, char **g_attrs
 
         tmp_dir = *dir;
 
+        /* When the maximum number of directories is reached, the excess is discarded and warned */
+        if (j++ >= MAX_DIR_SIZE){
+            mwarn(FIM_WARN_MAX_DIR_REACH, MAX_DIR_SIZE);
```

Comment:
Please print all records and directories that will not be monitored in the same message.